spec/features/user_logs_in_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+require 'support/authentication_helper'
+
+feature 'User logs in', %Q{
+  As an authenticated user
+  I want to sign in
+  So that I can track my medications and medical history
+} do
+
+#   Acceptance Criteria:
+# - There is a link to "log in"
+# - The log in page has a form for:
+#   - Email
+#   - Password
+# - The form requires me to provide all information
+# - I am logged in once i click "log in" button
+
+  scenario 'there is a link for the user to log in' do
+    visit root_path
+
+    expect(page).to have_content "Log In"
+  end
+
+  scenario 'log in form is provided' do
+    visit new_user_session_path
+
+    expect(page).to have_content "Email"
+    expect(page).to have_content "Password"
+  end
+
+  scenario 'user provides valid information' do
+    user = FactoryGirl.create(:user)
+    sign_in(user)
```

Comment:
It might be nice to do these sign-in steps manually for this test, just so that it's more explicit! That way, anyone who's looking at your codebase and wondering how the sign-in process works can come straight to this file and find all the answers here, instead of needing to look anywhere else. That's kind of a matter of opinion, though, so if you'd rather keep your code as-is that's fine too!